### PR TITLE
Add four new units relating to Electrical Power systems management.

### PR DESCRIPTION
**Pull request by JenTechSystems**
I need units relating to a few different Electrical Engineering concepts.

There has previously been talk of adding types for Apparent power ([PR #185]([url](https://github.com/anjdreas/UnitsNet/issues/185))). So I have added this as well as Reactive Power, Volts AC (ElectricalPotentialAc), Volts DC (ElectricalPotentialDc) and Siemens (ElectricAdmittance).

I have only added scales to each of these units which make sense in our domain as well as those that seem likely/reasonable. Obviously, this can be expanded when and as needed.

This is related to the issue I raised: https://github.com/anjdreas/UnitsNet/issues/221

And also to an older PR: https://github.com/anjdreas/UnitsNet/issues/185

### CODE REVIEW

File: UnitsNet/UnitDefinitions/AcElectricPotential.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "AcElectricPotential",
+  "BaseUnit": "VoltAc",
+  "XmlDoc": "The Electrict Potential of a system known to use Alternating Current.",
+  "Units": [
+    {
+      "SingularName": "VoltAc",
+      "PluralName": "VoltAcs",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/AcElectricPotential.json]: `VoltsAc` I think sounds better

File: UnitsNet/UnitDefinitions/DcElectricPotential.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "DcElectricPotential",
+  "BaseUnit": "VoltDc",
+  "XmlDoc": "The Electrict Potential of a system known to use Direct Current.",
+  "Units": [
+    {
+      "SingularName": "VoltDc",
+      "PluralName": "VoltDcs",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/DcElectricPotential.json]: Same here, `VoltsDc`.

File: UnitsNet/UnitDefinitions/ReactivePower.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "Name": "ReactivePower",
+  "BaseUnit": "VoltAmpereReactive",
+  "XmlDoc": "Volt-ampere reactive (var) is a unit by which reactive power is expressed in an AC electric power system. Reactive power exists in an AC circuit when the current and voltage are not in phase.",
+  "Units": [
+    {
+      "SingularName": "VoltAmpereReactive",
+      "PluralName": "VoltAmpereReactives",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/ReactivePower.json]: `VoltamperesReactive` as plural, I think?

File: UnitsNet/UnitDefinitions/ReactivePower.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+  "Name": "ReactivePower",
+  "BaseUnit": "VoltAmpereReactive",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/ReactivePower.json]: We have previously gone lower-case for words bound by hyphen `-`.
So `VoltampereReactive`.

File: UnitsNet/UnitDefinitions/AcElectricPotential.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "AcElectricPotential",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/AcElectricPotential.json]: I think rename to `ElectricPotentialAc` and `ElectricPotentialDc` below, so they units and json files get grouped together. Should help discoverability.

File: UnitsNet/UnitDefinitions/DcElectricPotential.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "DcElectricPotential",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/DcElectricPotential.json]: We should probably mark the existing `ElectricPotential` as obsolete when introducing this.

File: UnitsNet/UnitDefinitions/AcElectricPotential.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  "BaseUnit": "VoltAc",
+  "XmlDoc": "The Electrict Potential of a system known to use Alternating Current.",
```
Comment (JKSnd) [inline UnitsNet/UnitDefinitions/AcElectricPotential.json]: typo "Electrict"

File: UnitsNet/UnitDefinitions/DcElectricPotential.json (hunk 6 above)
Comment (JKSnd) [inline UnitsNet/UnitDefinitions/DcElectricPotential.json]: Agreed

File: UnitsNet/UnitDefinitions/ReactivePower.json (hunk 3 above)
Comment (JKSnd) [inline UnitsNet/UnitDefinitions/ReactivePower.json]: Agree

**Comment by JKSnd** [comment]
Will you have any need to deal with the phase of the AC quantities your software is dealing with?

If so, would introducing phase be outside the scope of this library? Maybe. More minds are better, lets discuss.

**Comment by JenTechSystems** [comment]
Hi @JKSnd 

Thanks for the forethought. We do not have a requirement to deal with the phase of the AC quantities.

File: UnitsNet/UnitDefinitions/AcElectricPotential.json (hunk 5 above)
Comment (JenTechSystems) [inline UnitsNet/UnitDefinitions/AcElectricPotential.json]: Agreed and changed.

File: UnitsNet/UnitDefinitions/AcElectricPotential.json (hunk 1 above)
Comment (JenTechSystems) [inline UnitsNet/UnitDefinitions/AcElectricPotential.json]: Agreed and changed.

File: UnitsNet/UnitDefinitions/DcElectricPotential.json (hunk 6 above)
Comment (JenTechSystems) [inline UnitsNet/UnitDefinitions/DcElectricPotential.json]: So from now on people will be forced to choose either ElectricPotentialAc or ElectricPotentialDc?

But there are many cases, in our software at least, where we don't care if its AC or DC...we just want to measure the Voltage.

File: UnitsNet/UnitDefinitions/DcElectricPotential.json (hunk 2 above)
Comment (JenTechSystems) [inline UnitsNet/UnitDefinitions/DcElectricPotential.json]: Agreed and changed.

File: UnitsNet/UnitDefinitions/ReactivePower.json (hunk 4 above)
Comment (JenTechSystems) [inline UnitsNet/UnitDefinitions/ReactivePower.json]: Agreed and changed.

File: UnitsNet/UnitDefinitions/ReactivePower.json (hunk 3 above)
Comment (JenTechSystems) [inline UnitsNet/UnitDefinitions/ReactivePower.json]: Agreed and changed.

File: UnitsNet/UnitDefinitions/AcElectricPotential.json (hunk 7 above)
Comment (JenTechSystems) [inline UnitsNet/UnitDefinitions/AcElectricPotential.json]: Fixed.

**Comment by anjdreas** [comment]
Alright! Latest changes look good, merged.

**Comment by anjdreas** [comment]
Nuget 3.51 out.

**Pull request merged**